xml/operating_system_darwin.xml

```diff
@@ -0,0 +1,110 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Patterns for common names of Mac OS X operating systems.
+-->
+<fingerprints matches="operating_system.name">
+  <fingerprint pattern="^Mac OS X\s?(\d+?(?:\.\d+?)+?)?$" flags="REG_ICASE">
+    <description>Mac OS X with version number</description>
+    <example os.version="10.10.5">Mac OS X 10.10.5</example>
```

Comment:
There is no support for negative tests.  I would not suggest putting this in rspec.  If you have a fingerprint you want a pattern #N to not match, add a pattern before #N that explicitly matches it;  order is enforced and tested.